apps/ffdweb-site/src/app/_components/PageHeader.tsx

```diff
@@ -0,0 +1,100 @@
+import Image from 'next/image'
+import type { StaticImageData } from 'next/image'
+
+import { Button } from '@filecoin-foundation/ui/Button'
+import { Heading } from '@filecoin-foundation/ui/Heading'
+import { buildImageSizeProp } from '@filecoin-foundation/utils/buildImageSizeProp'
+import type { CTAProps } from '@filecoin-foundation/utils/types/ctaType'
+import { clsx } from 'clsx'
+
+import { BASE_DOMAIN } from '@/constants/siteMetadata'
+
+type TitleProps = {
+  children: string
+  fontVariant?: PageHeaderFontVariant
+}
+
+export type StaticImageProps = {
+  data: StaticImageData
+  alt: string
+}
```

Comment:
What about the page header for `/blog` - Wouldn't it need a dynamic image? Same like FF?

```
type PageHeaderImageProps = (StaticImageProps | ImageProps) & {
  objectFit?: ImageObjectFit
}
```